tests/test_dates.py

```diff
@@ -20,29 +20,44 @@ def test_numeric_date(self):
         assert dates.numeric_date("1Y") == pytest.approx(1999.138, abs=1e-3)
         assert dates.numeric_date("1Y1M1W") == pytest.approx(1999.034, abs=1e-3)
 
-    def test_ambiguous_date_to_date_range_not_ambiguous(self):
-        assert dates.ambiguous_date_to_date_range("2000-03-29", "%Y-%m-%d") == (
-            datetime.date(year=2000, month=3, day=29),
-            datetime.date(year=2000, month=3, day=29),
+    def test_get_numerical_date_from_value_not_ambiguous(self):
+        assert (dates.get_numerical_date_from_value("2000-03-29", "%Y-%m-%d")
+            == pytest.approx(dates.numeric_date(datetime.date(year=2000, month=3, day=29)), abs=1e-3)
```

Comment:
Cool! I didn't know about `pytest.approx` before, but it looks really useful. (I see now it was in the code base previously.)